src/main/resources/messages_template.xml

```diff
@@ -333,6 +333,11 @@ See the accompanying LICENSE file for applicable license.
     <response></response>
   </message>
 
+  <message id="DOTJ070I" type="INFO">
+    <reason>Invalid 'class' attribute '%1' for was found for element '&lt;%2&gt;'.</reason>
+    <response>This generally indicates that your DTD or Schema was not developed properly according to the DITA specification.</response>
```

Comment:
Agreed. FWIW I made this by copying `DOTJ030I` which is a similar condition and has same response. Should probably fix both since this PR already works with both.